auth/utils.py

```diff
@@ -0,0 +1,15 @@
+import jwt
+
+SECRET_KEY = '1234567890'  # TODO: Move to environment variables
+
```

Comment:
read from env and save. to variable
```suggestion
import os

SECRET_KEY = os.getenv('SECRET_KEY', 'default_secret_key')
```